packages/@sanity/client/README.md

```diff
@@ -32,14 +32,22 @@ const client = sanityClient({
 
 Initializes a new Sanity Client. Required options are `projectId` and `dataset`.
 
-### Fetch a single document
+### Fetch an entire single document
 
 ```js
 client.getDocument('bike-123').then(bike => {
   console.log(`${bike.name} (${bike.seats} seats)`)
 })
 ```
 
+### Fetch multiple entire documents in one go
+
+```js
+client.getDocuments(['bike123', 'bike345']).then(([bike123, bike345]) => {
+  console.log(`${bike.name} (${bike.seats} seats)`)
+})
+```
```

Comment:
We should probably document that missing documents are returned as `null`-values, and that they are ordered in the same order as the input IDs (I'm used to thinking in SQL `SELECT * FROM documents WHERE id IN (a, b, c)`)

Reply:
Good point! Guess I assumed that the order was preserved apart from the omitted null values.

Reply:
Noted this in the README now, linking to the official doc endpoint docs. Also moved the `getDocument(s)` sections from the top of the README to below the _queries_ section to make it less prominent/significant in the README.